autoload/widgets/fzf-named-directories

```diff
@@ -0,0 +1,8 @@
+#!/bin/zsh
+
```

Comment:
Add `__fzf::widget::init 'cd --' || return 1`

Reply:
```sh
__fzf::widget::init 'cd --' || return 1
```

but then it cannot be used inside the command line, e.g.

```sh
cp file <insert-named-directory-here>
```

Reply:
In that case, create `fzf-insert-named-directories`.

Reply:
Please refer to following widgets.
* [fzf-insert-directory](https://github.com/ytet5uy4/fzf-widgets/blob/master/autoload/widgets/fzf-insert-directory)
* [fzf-insert-files](https://github.com/ytet5uy4/fzf-widgets/blob/master/autoload/widgets/fzf-insert-files)

Reply:
If you create `fzf-insert-named-directories` and send such PR, I merge it.